setup.py

```diff
@@ -13,16 +13,17 @@
 
 requirements = [
```

Comment:
can't this be taken automatically from `requirements.txt` ? or what's the usual way of doing this?